x/gov/keeper/deposit.go

```diff
@@ -185,3 +185,32 @@ func (keeper Keeper) RefundDeposits(ctx sdk.Context, proposalID uint64) {
 		return false
 	})
 }
+
+// from sdk 47 - backport modified for Dawn/Terra Classic
+
+// validateInitialDeposit validates if initial deposit is greater than or equal to the minimum
+// required at the time of proposal submission. This threshold amount is determined by
+// the deposit parameters. Returns nil on success, error otherwise.
+func (keeper Keeper) validateInitialDeposit(ctx sdk.Context, initialDeposit sdk.Coins) error {
+	params := keeper.GetDepositParams(ctx)
+
+	// hardcoded here, as requested by Zaradar
+	minInitialDepositRatio := sdk.NewDecWithPrec(1, 2)
+
+	//if err != nil {
+	//	return err
+	//}
+
+	// Not needed - as hardcoded
```

Comment:
I think you should re-enable this logic. We will probably need it down the line and technically we have no way of knowing if NewDecWithPrec would somehow return a zero value in the future, which means we might as well guard against this case considering the saving in terms of milliseconds from not doing this check is probably negligible? :)